runhouse/resources/function.py

```diff
@@ -140,6 +149,13 @@ def to(
 
         return new_function
 
+    @property
+    def __name__(self):
```

Comment:
I dont think we should overwrite the behavior of name (this is initialized in parent classes), and we also do want to allow anonymous functions

Reply:
Yeah agreed, deleted it, it was part of the debugging I was doing.